dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/InDevelopmentPavedRoadFramework.kt

```diff
@@ -51,7 +57,7 @@ abstract class InDevelopmentPavedRoadFramework(
 
         @Suppress("TooGenericExceptionCaught")
         try {
-            viewConfig.build(into = datalandProject)
+            viewConfig.build(into = datalandProject, privateFrameworkBoolean)
```

Comment:
Was wird bei der viewConfig anders gemacht für private frameworks?
Hätte erwartet, dass die ViewConfig selbst (als File) unabhängig von private oder public immer gleich aussieht.

Reply:
Ahh ok habs jetzt verstanden. In `viewConfig.build()` wird noch mehr generiert als "nur" das ViewConfig.ts file.  Da werden auch der ApiClient und die Registry Einträge gehandlet, so wie ich das verstanden habe.

Deswegen gibst du also auch die flag mit.

Aber Frage/Vorschlag: Ist es dann nicht irreführend, dass das Ding "FrameworkViewConfigBuilder" heißt?
Ist es dann nicht sogar noch mehr, und zwar ein "FrontendCodeBuilder" ?

Reply:
Stimme ich zu, dass er mehr macht als man vermutet. Er baut zum Beispiel nicht die Uploadpage, daher fände ich FrontendCodeBuilder auch nciht ganz treffend.
Wir können es erstmal stehen lassen, wenn eine bessere Bezeichnung einfällt und, dass nicht zu viele Files betrifft würde ich es dann ändern.